include/wx/debugrpt.h

```diff
@@ -131,7 +132,9 @@ class WXDLLIMPEXP_QA wxDebugReport
 class WXDLLIMPEXP_QA wxDebugReportCompress : public wxDebugReport
 {
 public:
-    wxDebugReportCompress() { }
+    // pass a non-empty string to specify a custom temporary directory to be
+    // used for report construction
+    wxDebugReportCompress(const wxString &tmpDir = wxEmptyString);
```

Comment:
Ctor should be made `explicit` (same applies to the base `wxDebugReport`, now that I think about it).